src/components/route-search/ListRequestSection.tsx

```diff
@@ -10,13 +10,15 @@ const ListRequestSection = () => {
       <MainTitle>찾으시는 리스트가 없나요?</MainTitle>
       <SubTitle>필요한 상황의 리스트를 말씀해 주세요.</SubTitle>
       <StyledButton>
-        <div>
-          <Emoji role="img" aria-label="writing hand">
-            ✍
-          </Emoji>
-          리스트 요청하기
-        </div>
-        <IconChevron24pxRightLeft />
+        <StyledAnchor href="https://tally.so/r/w5Xqyd" target="_blank">
```

Comment:
```suggestion
        <StyledAnchor href="https://tally.so/r/w5Xqyd" target="_blank" rel="noreferrer">
```

https://stackoverflow.com/questions/50709625/link-with-target-blank-and-rel-noopener-noreferrer-still-vulnerable

https://velog.io/@neori/HTML-a%EC%97%90-targetblank%EC%99%80-relnoreferrer-noopener%EB%A5%BC-%ED%95%A8%EA%BB%98-%EC%82%AC%EC%9A%A9%ED%95%B4%EC%95%BC-%ED%95%98%EB%8A%94-%EC%9D%B4%EC%9C%A0

최신 브라우저에서는 `noopener`가 기본적으로 붙어있다고 하네요 !!

https://chromestatus.com/feature/6140064063029248

Reply:
오 좋은 정보 감사합니다!!! 👍🏻